conda_build/noarch_python.py

```diff
@@ -103,7 +117,7 @@ def transform(m, files):
     _force_dir(scripts_dir)
 
     # Create windows prelink script
-    with open(join(scripts_dir, '.%s-pre-link.bat' % name), 'w') as fo:
+    with open(join(scripts_dir, '.%s-pre-link.bat' % name), 'w', newline='\r\n') as fo:
```

Comment:
On Windows, it would be `\r\n` by default. But when building on Linux, it would be cleaner if it still produced a .bat with Windows newlines, IMO.